packages/core/src/storybook/stand-alone-documentaion/spacing/spacing.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import { SpacingSizes } from "./spacing-sizes/spacing-sizes";
+import { StoryObj , Meta } from "@storybook/blocks";
+
+const meta: Meta = {
+    title: "Foundations/Spacing"
+  };
+  export default meta;
+  
+  type Story = StoryObj;
+  
+  export const Sizes: Story = {
+    render: SpacingSizes
+  };
```

Comment:
can you please run prettier on this file? it is missing a new line at the end of the file